<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1439</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_first_img">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>820</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_mask_folder">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prev_img">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>820</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textedit_bgImgFolder">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_img">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>820</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cnt">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>830</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_randomBgImgs">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>610</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Generate Random Background Images</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_bg_folder">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>300</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Background Images Folder:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cropped Part Mask and Img Folder path:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textedit_maskImgFolder">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>650</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_display">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>651</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_last_img">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>820</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cropOutEachPart">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>110</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(86, 56, 36);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>CropOut EachPart</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>1251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textedit_fullImg_maskImgFolder">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>650</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Full Img Mask and Img Folder path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse_fullImgmask_folder">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>40</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(112, 133, 255);
	border-radius: 20px;
}

QPushButton:hover {
	background-color: rgb(110, 33, 255);
	color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>420</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Number of Images to generate</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_nImgToGenerate">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>410</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_nMaxPartsPerImg">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>480</y>
     <width>104</width>
     <height>41</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>490</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Maximum No of parts per Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
